<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #152028;
color: #efefef;
QLineEdit {
 border: 2px solid gray;
 border-radius: 10px;
}</string>
  </property>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>331</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #1F2933;
border: 2px solid gray;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="AddLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-style:italic;&quot;&gt;CryptoVault&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #152028;
color: #efefef;
border: #1F2933;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UsernameInput">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #152028;
color: #efefef;
border: 2px solid #1F2933;
border-radius: 15px;
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>   Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EncryptionKeyInput">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #152028;
color: #efefef;
border: 2px solid #1F2933;
border-radius: 15px;
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>   Password</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
